### PR TITLE
Remove Qt3Support warnings (group 6)

### PR DESCRIPTION
**Pull request by tom-perkins**
This is group 6 of #16128.

(While in this section, also replaced some dead links with links to MantidPlot's documentation.)

**To test:**

Code review, check MantidPlot builds and looks right - particularly context menus for graphs and tables.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the coding standards? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by KarlPalmen** [comment]
Anton has provided a test description in #16305 , which I think applies here too, so I'll follow it. I'm grateful for the short branch name.

**Comment by KarlPalmen** [comment]
Which parts of Mantid plot are affected by your changes?

**Comment by KarlPalmen** [comment]
I see you mention "particularly context menus for graphs and tables".

**Comment by tom-perkins** [comment]
The changes are in:
- Graph context menu (right-click)
- Window context menu
- Table context menu
- Help (if help files are not found - unlikely to happen)
- Add function curve to graph
- "Plot 3D" tool bar

**Comment by KarlPalmen** [comment]
Thank you for the information.

**Comment by KarlPalmen** [comment]
Testing has gone well for me, except possibly for the 3D tool bar, which I may have not tested well. So with that caveat :shipit: .

File: MantidPlot/src/ApplicationWindow.cpp [hunk 1]
```diff
@@ -10478,17 +10478,17 @@ void ApplicationWindow::showStandAloneHelp() {
...
-                          tr("The manual can be downloaded from the following "
+                          tr("The manual can be found at the following "
                              "internet address:") +
                               "<p><a href = "
-                              "http://soft.proindependent.com/"
-                              "manuals.html>http://soft.proindependent.com/"
-                              "manuals.html</a></p>");
+                              "http://www.mantidproject.org/"
+                              "MantidPlot:_Help>http://www.mantidproject.org/"
+                              "MantidPlot::_Help</a></p>");
```
Comment (SimonHeybrock) [inline MantidPlot/src/ApplicationWindow.cpp]: Is this one colon too many? It seems to be only the link text, but if someone tries to copy and paste that?

Comment (tom-perkins) [inline MantidPlot/src/ApplicationWindow.cpp]: You're right, I think that's a typo. I'll push a fix for this. 

(On looking closer, though, I can't find anywhere in the code where this function is called - it should probably just be deleted.)

Comment (SimonHeybrock) [inline MantidPlot/src/ApplicationWindow.cpp]: @tom-perkins Removing sounds good, the compiler will tell us if it is correct.

Comment (tom-perkins) [inline MantidPlot/src/ApplicationWindow.cpp]: Sounds good, I've added a new issue #16326 to do this and look for other functions that aren't used any more.

Comment (SimonHeybrock) [inline MantidPlot/src/ApplicationWindow.cpp]: :+1:
I will merge this once the build passed.

**Pull request merged**